### PR TITLE
Use GitHub Actions + Dependabot

### PR DESCRIPTION
**Pull request by bkeepers**
- [x] Add CI workflow for actions
- [x] Remove Travis config
- [x] Update main.workflow to use yml syntax
- [x] Add dependabot config

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+on:
+  push:
+    branches: [master]
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: ['2.5', '2.6', '2.7']
```
Comment (cadwallion) [inline .github/workflows/ci.yml:11]: Do we want to add 3.0 here as well?

**Review by cadwallion — APPROVED** [review]
👍🏻

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bkeepers) [reply .github/workflows/ci.yml:11]: Definitely! I tried adding it, but there are some dependencies that aren’t compatible. Would it be cool to do that in a follow up PR?

Comment (cadwallion) [reply .github/workflows/ci.yml:11]: Ah, ok, thanks for the context!  I'm fine with that being a separate PR then.

**Pull request merged**